aeriscloud/cli/cloud/organization.py

```diff
@@ -201,33 +198,58 @@ def init(name, repository):
-            config.has('github', 'token')):
-        gh = Github()
-
-        if '/' in name:
-            split = name.split('/')
-            name = split[0]
-            repo_name = '-'.join(split[1:]) + "-aeriscloud-organization"
+    dirname = name
+    # If remote is not specified
+    if not repository:
+        # If GH integration is enabled
+        if has_github_integration():
+            gh = Github()
+
+            if '/' in name:
+                split = name.split('/')
+                name = split[0]
+                repo_name = '-'.join(split[1:]) + "-aeriscloud-organization"
+                dirname = '-'.join(split)
+            else:
+                repo_name = "aeriscloud-organization"
+
+            # If member of the organization
+            orgs = [org for org in gh.get_organizations()
+                    if org.login.lower() == name.lower()]
+            if orgs:
+                # If repo exists
+                repos = [repo.name for repo in orgs[0].iter_repos()
+                         if repo.name.lower() == repo_name.lower()]
+                if not repos:
+                    # Give instructions to create the repo
+                    info("""The repository {repo} has not been found in the {org} organization.
+You can create a new repo at the following address: https://github.com/new."""
+                         .format(repo=repo_name, org=name))
+                    if not click.confirm("Do you want to continue?",
+                                         default=False):
+                        info("Aborted. Nothing has been done.")
+                        return
+            # If not member of the organization
+            else:
+                warning("We were not able to verify the existence of the "
+                        "repository as you don't belong to the {org} "
+                        "organization.".format(org=name))
+                if not click.confirm("Do you want to continue?",
+                                     default=False):
+                    info("Aborted. Nothing has been done.")
+                    return
+
+            repository = "git@github.com:{org}/{repo}.git".format(
+                org=name,
+                repo=repo_name
+            )
         else:
-            repo_name = "aeriscloud-organization"
-
-        orgs = [org for org in gh.get_organizations()
-                if org.login.lower() == name.lower()]
-        if orgs:
-            repos = [repo.name for repo in orgs[0].iter_repos()
-                     if repo.name.lower() == repo_name.lower()]
-            if repos:
-                repository = "git@github.com:{org}/{repo}.git".format(
-                    org=name,
-                    repo=repo_name
-                )
+            fatal("You need to specify a repository URL or enable the GitHub "
+                  "integration in AerisCloud.")
 
-    dest_path = get_env_path(name)
+    dest_path = get_env_path(dirname)
```

Comment:
You don't know the name of the folder unless you do the github checks.

I could assume that it follows the GH pattern when we have a `/` in the same, and compute the dirname before doing anything with GH.